.github/workflows/build_images.yml

```diff
@@ -0,0 +1,80 @@
+name: build images
```

Comment:
```suggestion
# SPDX-FileCopyrightText: 2022 Helmholtz Centre for Environmental Research GmbH - UFZ
#
# SPDX-License-Identifier: EUPL-1.2

name: build images
```

Would you mind adding this or are there any other constraints that would require another license?